shenyu-examples/shenyu-examples-http/pom.xml

```diff
@@ -35,6 +35,11 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
```

Comment:
why add this

Reply:
This dependency is required for file upload and download in this way. But this approach will conflict with the existing code, so I'm going to try to write it with weblux